### PR TITLE
Darkmode/course-feature-bubbles

### PR DESCRIPTION
**Pull request by K02D**
Course section bubbles were broken on light mode too so I fixed them

Before:
image

image

After:
image

image

### CODE REVIEW

File: static/css/timetable/framework/page_layout.scss [hunk 1]
```diff
@@ -10,7 +10,6 @@ h5,
 h6,
 p,
 a,
-span,
```
Comment (K02D) [inline static/css/timetable/framework/page_layout.scss:13]: For some reason, all span elements were set to [this](https://github.com/jhuopensource/semesterly/blob/52605bb9531f83547947ebc4853e84f9015a23a9/static/css/timetable/base/themes.scss#L83) blue color. This seemed

File: static/css/timetable/modules/pagination.scss [hunk 2]
```diff
@@ -54,9 +56,17 @@
   &.active {
     background-color: $primary-navy;
     border: 5px solid $primary-navy;
-    color: $gfff;
+    span.section-bubble {
```
Comment (K02D) [inline static/css/timetable/modules/pagination.scss:59]: To override the existing style I increased the selector specificity

File: static/css/timetable/modules/pagination.scss [hunk 3]
```diff
@@ -54,9 +56,17 @@
...
+      @include theme() {
+        color: t("bubble-text-color");
+      }
+    }
```
Comment (K02D) [inline static/css/timetable/modules/pagination.scss:63]: To override the existing style I had to give a higher selector specificity (span.section-bubble)

**Review by JJamesWWang — APPROVED** [review]
잘 보여요

**Pull request merged**